sdcm/nemesis/monkey/abort_decommission.py

```
@@ -91,14 +92,26 @@ def disrupt(self):
-        # Run decommission and abort in parallel, since we want to abort as soon as streaming starts, without waiting for decommission to finish
-        ParallelObject(
-            objects=[self.decommission_target_node, self.abort_decommission_task], timeout=600
-        ).call_objects()
+        try:
+            # Run decommission and abort in parallel, since we want to abort as soon as streaming starts, without waiting for decommission to finish
+            ParallelObject(
+                objects=[self.decommission_target_node, self.abort_decommission_task], timeout=600
+            ).call_objects()
+        except KillNemesis:
+            return
+        except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)